backend/services/mgmt/v1alpha1/connection-data-service/connection-data.go

```diff
@@ -627,7 +627,7 @@ func (s *Service) GetConnectionSchema(
 					return nil, err
 				}
 				if out == nil {
-					break
+					continue
```

Comment:
maybe we should add a nice log statement here ? It might be nice if it's an edge case.